editoast/src/views/timetable/stdcm.rs

```diff
@@ -381,6 +389,7 @@ impl VirtualTrainRun {
         stdcm_request: &Request,
         infra: &Infra,
         rolling_stock: &RollingStockModel,
+        consist_parameters: &PhysicsConsistParameters,
```

Comment:
Since a `PhysicsConsistParameters` contains a `RollingStock` we should only provide the consist.